code/src/components/MessageList.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import moment from "moment";
+
+export const MessageList = ({ messageList, postHearts }) => {
+  return (
+    <div>
+      {messageList.map((message) => (
+        <article>
```

Comment:
missing `key` prop over here. It should always be added to root element of loop, in your case it's `article`, not `p`

Reply:
👌🏽fixed it, now its like this <article key={message._id}>